srcDoc/release_notes.rst

```diff
@@ -334,6 +334,18 @@ This makes the function OS independent.
 At the same time, the code has been improved avoid overflowing the time for the sleeping
 period.
 
+Incorrect time stamp using micro on windows (#202)
```

Comment:
I don't think we need a public release note for this, as micro was never launched officially, so you can skip this.

Reply:
Remove it.